jaas/src/main/java/com/griddynamics/jagger/jaas/service/impl/TestEnvironmentServiceImpl.java

```diff
@@ -100,7 +94,7 @@ else if (!isEqualCollection(testEnvToUpdate.getLoadScenarios(), newTestEnv.getLo
         }
 
         if (testEnvToUpdate.getRunningLoadScenario() != newTestEnv.getRunningLoadScenario()) {
-            testEnvToUpdate.setRunningLoadScenario(getNewRunningLoadScenario(newTestEnv, testEnvToUpdate));
+            testEnvToUpdate.setRunningLoadScenario(newTestEnv.getRunningLoadScenario());
```

Comment:
Something like this. But, if I'm not mistaken, you cannot add new load scenario to test env and set it as runing in one transaction... so, maybe, you'll need to make 2 updates

`
  if (!testEnvToUpdate.getLoadScenarios().contains(newTestEnv.getRunningLoadScenario()))
            testEnvToUpdate.getLoadScenarios().add(newTestEnv.getRunningLoadScenario());
`